pydy/codegen/ode_function_generators.py

```diff
@@ -169,8 +169,8 @@ def _deduce_system_type(**kwargs):
         full mass matrix
             M(x, p) * x' = f(x, t, r, p)
         min mass matrix
-            M(q, p) * u' = f(q, u, t, r, p)
-            q' = g(q, u, t)
+            M(x, p) * x'_d = f(x, t, r, p)
+            x'_k = g(x, t)
```

Comment:
Great, I'll work on the notation.

For the case with an A matrix and g vector, won't most solvers take advantage of the block-matrix nature if M and A were combined?

Do you have an example of a System generating the minimum mass matrix form?